source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Custom Certificates on the Global CDN
```

Comment:
Suggest a title similar to the doc on free HTTPS, such as: 
> Custom HTTPS on Pantheon's Global CDN

Reply:
OR
> Early Access: Custom HTTPS on Pantheon's Global CDN

Reply:
"Custom Certificates on Pantheon's Global CDN" sounds good. HTTPS isn't custom as it's a universal TLS configuration.

Reply:
defer to @ari-gold, i'm nit picking for sure.. "custom certificates" seem like a less obvious alternative to "free & managed https"-- why not just say we have 2 types of https implementations to pick from? free / managed or custom? but yeah, i hear you. https itself isn't custom.

Reply:
Good point. We could re-frame "free & managed HTTPS" as something like "Automated Certificate Management" vs "Manually Managed Custom Certificates" but let's circle back on that and focus on shipping MVP

Reply:
so, tl;dr is that the title stays as is for now?

Reply:
No, Ari's last suggestion was "Custom Certificates on Pantheon's Global CDN" I think that's good. In the past we've added "Early Access:" to the title to signal availability. Advantage for that is signaling folks that miss the notice by mistaking it for an ad. Defer to ya'll